### ISSUE TITLE
Improve package search

### ISSUE THREAD

**Issue by bjarnef** [umbraco/OurUmbraco#796]
## Detailed description

I always found the package search a bit strange, because it often doesn't find packages where the package name contain the exact search term, e.g. search for "gmaps" on https://our.umbraco.com/packages/

![image](https://user-images.githubusercontent.com/2919859/190977217-713b500d-10d3-477b-92d4-fa102d1233f6.png)

when searching for "Our.Umbraco.Gmaps" the package is found, but I would probably expect it to be weighted higher.

**Comment by abjerner** [comment]
👍 

I think being able to find **Our.Umbraco.Gmaps** when searching for **gmaps** will require enabling leading wildcards. Or that **Our.Umbraco.Gmaps** is also indexed as **Our Umbraco Gmaps**.

This is the main reason I started giving our packages friendly names: https://our.umbraco.com/packages/website-utilities/skybrud-redirects/

**Comment by bjarnef** [comment]
@abjerner yes, indexing the name "Our.Umbraco.Gmaps" and separate words "Our Umbraco Gmaps" would enhance the search. Alternatively it could be useful to allow package developers to as tags/keywords to a project, either a fancy tags editor or just a text input and separate keywords by comma and include these in the index.

For example you may want to include tags "301" and "302" for the [Skybrud Redirects package](https://our.umbraco.com/packages/website-utilities/skybrud-redirects/).

Currently it isn't found when searching "301" https://our.umbraco.com/search?q=301&cat=project but you could probably include "301" and "302" somewhere in the description 😉

**Comment by abjerner** [comment]
It sounds like Umbraco is in the progress of splitting up Our into different pieces, with an Umbraco Marketplace to replace the package piece. I don't really now how far Umbraco is in releasing this, but it probably means that we shouldn't put too much work into the current packages section of Our.

But with that being said, it should be possible to optimize how packages are indexed in Umbraco.

And yes, I should probably mentions a few keywords on the package page 👍

**Comment by nul800sebastiaan** [comment]
FYI: I don't recommend spending any time on this at the moment, a new marketplace is under development that will replace both the packages section and the current marketplace.

I have no timeline for when it will be released but it feels like it an investment in something that has been broken for so many years wouldn't make much sense.

**Comment by bjarnef** [comment]
@nul800sebastiaan anywhere we could follow the progress on the new marketplace or see it in beta (before final launch).
Not sure how much the community is involved in this process, but I am sure many would love to give feedback 😁

**Comment by nul800sebastiaan** [comment]
I know the packages team is giving feedback, no idea what the rest of the plans are though.
